attachment.py

```diff
@@ -25,13 +25,18 @@ class Attachment(metaclass=PoolMeta):
     can_see_signatures = fields.Function(
         fields.Boolean('Can See Signatures'),
         'on_change_with_can_see_signatures')
+    can_force_signature = fields.Function(
+        fields.Boolean('Can Force Signature'),
+        'on_change_with_can_force_signature')
 
     @classmethod
     def __setup__(cls):
         super(Attachment, cls).__setup__()
         cls._buttons.update({
                 'init_new_signature_process': {
-                    'invisible': ~Eval('can_create_new_signature')}
+                    'invisible': ~Eval('can_create_new_signature')},
+                'manually_create_forced_signature': {
+                    'invisible': ~Eval('can_force_signature')}
```

Comment:
Style : "," en fin de ligne

```suggestion
                    'invisible': ~Eval('can_force_signature')},
```